File: js/1672_Richest_Customer_Wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let maxSum = 0;
    let tempSum = 0;
    accounts.forEach((account) => {
        account.forEach((price) => {
            tempSum += price;  
        });
        maxSum = Math.max(maxSum, tempSum);
        tempSum = 0;
    });
    
    return maxSum;
};

File: java/1672_Richest_Customer_Wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maxSum = 0;
        int tempSum = 0;
        for (int[] account : accounts) {
            for (int price : account) {
                tempSum += price;
            }
            maxSum = Math.max(maxSum, tempSum);
            tempSum = 0;
        }

        return maxSum;
    }
}
